Brain Blocks

```diff
@@ -1,4 +0,0 @@
-Front
-Back
```

Comment:
Interprets language, words
Interprets signals from vision, hearing, motor, sensory, and memory
Interprets vision (color, light, movement)
Spatial and visual perception